### PR TITLE
NO-JIRA Fixing the paths for the Docker examples

### PR DESCRIPTION
**Pull request by chrisbier**

### CODE REVIEW

**Comment by jbertram** [comment]
Can you squash your 2 commits down to just 1?

**Comment by jbertram** [comment]
And also add "NO-JIRA" to the beginning of the commit message?

**Comment by chrisbier** [comment]
Sure thing

File: artemis-docker/Dockerfile-centos [hunk 1]
```diff
@@ -57,7 +57,7 @@ USER root
 
 RUN mkdir /var/lib/artemis-instance && chown -R artemis.artemis /var/lib/artemis-instance
 
-COPY ./docker/docker-run.sh /
+COPY ./artemis-docker/docker-run.sh /
```
Comment (brusdev) [inline artemis-docker/Dockerfile-centos:60]: This path don't match the path created by the `prepare-docker.sh` script: https://github.com/apache/activemq-artemis/blob/2.15.0/artemis-docker/prepare-docker.sh#L54

Comment (chrisbier) [reply artemis-docker/Dockerfile-centos:60]: When I ran it on my machine, I had to fix the path in order for the Dockerfile to find the file.
```Step 15/20 : COPY ./docker/docker-run.sh /
COPY failed: stat /var/lib/docker/tmp/docker-builder613788742/docker/docker-run.sh: no such file or directory
```

Comment (chrisbier) [reply artemis-docker/Dockerfile-centos:60]: Ah. Sorry! My fault for not reading closely.

**Pull request closed without merging**